app/src/main/java/com/sopt/now/compose/presentation/home/HomeViewModel.kt

```diff
@@ -0,0 +1,39 @@
+package com.sopt.now.compose.presentation.home
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.sopt.now.compose.data.ServicePool
+import com.sopt.now.compose.model.Friend
+import com.sopt.now.compose.model.User
+import com.sopt.now.compose.util.UiState
+import kotlinx.coroutines.launch
+
+class HomeViewModel : ViewModel() {
+    val mockUserList = listOf(
+        User(
+            id = "test",
+            pw = "test",
+            nickname = "기먼지",
+            tel = "010-2002-0625"
+        )
+    )
+
+    private val _getUserList = MutableLiveData<UiState<List<Friend>?>>(UiState.Loading)
+    val getUserList: MutableLiveData<UiState<List<Friend>?>> = _getUserList
```

Comment:
LiveData로 상태에 대한 선언을 잘 해주셨네요! 아래 두가지만 참고해주세요 👍 
- 지금 변수는 상태에 가깝기 떄문에 특정 행위를 하는 `get~` 형식의 이름보다는 friendUiState로 해보시면 어떨까요? 또한 이미 `getUserList`라는 함수가 있어서 혼동하기 쉬울 것 같기도 해요.
- 추가로 외부로 노출되는 `getUserList`는 MutableLiveData가 아닌 LiveData로 변할 수 없도록 설정해주시면 좋을 것 같아요. 

```suggestion
    private val _friendUiState = MutableLiveData<UiState<List<Friend>?>>(UiState.Loading)
    val friendUiStateL: LiveData<UiState<List<Friend>?>> = _getUserList.asLiveData()
```

Reply:
추후 sharedFlow로 변경할 예정이었는데 같이 고려해서 수정하겠습니다! 저는 해당 api 메서드로 네이밍을 짓는 편이었는데 ~UiState 형식으로 네이밍 짓는 것도 좋아보이네요! 천천히 수정해보도록 하겠습니닷 ㅎㅎ